data/ignition/files/usr/local/bin/start-agent.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+source common.sh
+
+echo "Waiting for infra-env-id to be available"
```

Comment:
Better to have the messages in stderr
```suggestion
>&2 echo "Waiting for infra-env-id to be available"
```